### DISCUSSION TITLE
Suggest for Chinese Traditional Support

### DISCUSSION THREAD

**Discussion by TragicLifeHu** [AppGround-io/bluetooth-keyboard-and-mouse-support#202] [Ideas]
As an user of this app, I found there's no any support for translation, keyboard layout and etc for Chinese, both Simplified (zh-CN) and Traditional (zh-TW). I would like to help translating into Traditional Chinese (Arrays and strings XML file needed); and for Chinese Traditional's Zhu'yin layout, I wish the devs could help.

**Comment by KiruyaMomochi** [comment]
Chinese keyboards are usually in US layout with/without Chinese input method labels printed on keys. **Without an input method handler activated, these keyboards would simply respond to Latin characters as physically labelled.** 
> See [Wikipedia](https://en.wikipedia.org/wiki/Keyboard_layout#Chinese) for more information.

And from our virtual keyboard side, it may not be possible to send *Bopomofo* (Or *Zhuyin*, but not *Zhu'yin* please) or Cangjie characters directly. Instead, we need to activate IME for the remote device.

> There is a workaround: use [Alt Code](https://en.wikipedia.org/wiki/Alt_code) or similar things. But this is platform-dependent and is not granted to work on all devices.

> Another workaround on user's side: use a special IME (may customize with [TRIME](https://github.com/osfans/trime)), which when you type Bopomofo or Cangjie characters, it actually types the corresponding latin character.

> Another workaround on developer's side: allow user to create custom mapping between unsupported Unicode keys and keystrokes.

**Comment by ywtong** [comment]
@developer is it possible to support Chinese language in the near future?
